charts/scalardb/templates/envoy/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "scalardb.fullname" . }}-envoy
+  labels:
+    {{- include "scalardb-envoy.labels" . | nindent 4 }}
+spec:
+  minReadySeconds: 0
+  replicas: {{ .Values.envoy.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "scalardb-envoy.selectorLabels" . | nindent 6 }}
+  {{- with .Values.envoy.strategy }}
+  strategy:
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+  template:
+    metadata:
+      labels:
+        {{- include "scalardb-envoy.selectorLabels" . | nindent 8 }}
+    spec:
+    {{- with .Values.envoy.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      securityContext:
+        {{- toYaml .Values.envoy.podSecurityContext | nindent 8 }}
+      restartPolicy: Always
+      terminationGracePeriodSeconds: 60
+      containers:
+        - name: {{ .Chart.Name }}-envoy
+          securityContext:
+            {{- toYaml .Values.envoy.securityContext | nindent 12 }}
+          image: "{{ .Values.envoy.image.repository }}:{{ .Values.envoy.image.version }}"
+          imagePullPolicy: {{ .Values.envoy.image.pullPolicy }}
+          ports:
+          - containerPort: 50051
+          - containerPort: 9001
```

Comment:
Helm Chart looks fine, but there is a slight difference in Deployment containerPort compared to scalardl. Is this a problem?

```
$ grep -r containerPort scalardb/templates/envoy
scalardb/templates/envoy/deployment.yaml:          - containerPort: 50051
scalardb/templates/envoy/deployment.yaml:          - containerPort: 9001

$ grep -r containerPort scalardl/templates/envoy
scalardl/templates/envoy/deployment.yaml:          - containerPort: 50051
scalardl/templates/envoy/deployment.yaml:          - containerPort: 50052
scalardl/templates/envoy/deployment.yaml:          - containerPort: 9001
```